docs/text.md

```diff
@@ -493,7 +493,7 @@ default is `true`.
 
   - **`fontVariant`**: array of enum('small-caps', 'oldstyle-nums', 'lining-nums', 'tabular-nums', 'proportional-nums') (_iOS_)
 
-  - **`letterSpacing`**: number (_iOS_)
+  - **`letterSpacing`**: number
```

Comment:
It's probably prudent to amend this, perhaps to say something like `(iOS, Android >= 5.0)`.